cmd/node/app_test.go

```diff
@@ -168,16 +164,37 @@ func (suite *AppTestSuite) TestMultipleNodes() {
 	}
 	txbytes, _ := types.SignedTransactionAsBytes(tx)
 	path := "../tmp/test/state_" + time.Now().String()
+
 	genesisTime := time.Now().Add(20 * time.Second).Format(time.RFC3339)
-	suite.initMultipleInstances(5, path, genesisTime)
+
+	poetClient, err := NewRPCPoetHarnessClient()
+	if err != nil {
+		log.Panic("failed creating poet client harness: %v", err)
+	}
+	suite.poetCleanup = poetClient.CleanUp
+
+	rolacle := eligibility.New()
+	rng := BLS381.DefaultSeed()
+
+	suite.initMultipleInstances(rolacle, rng, 5, 6, path, genesisTime, poetClient)
 	for _, a := range suite.apps {
 		a.startServices()
 	}
 
+	activateGrpcServer(suite.apps[0])
+
+	startInLayer := 9 // delayed pod will start in this layer
+	go func() {
+		delay := float32(suite.apps[0].Config.LayerDurationSec) * (float32(startInLayer) + 0.5)
+		<-time.After(time.Duration(delay) * time.Second)
```

Comment:
I'd consider using `genesisTime` to calculate the absolute time to start the additional nodes and wait until then. Note that you're currently starting to count 20 seconds too early.

Reply:
```suggestion
		<-time.Until(genesisTime.Add(time.Duration(delay) * time.Second))
```

Reply:
I took the thumbs up away since it is not right. The late node joins after genesis and uses the same genesisTime in his configuration.

Reply:
**_You're_** not right 😝 (as discussed in person)